tests/kvbm/test_determinism_disagg.py

```
@@ -0,0 +1,468 @@
+        while time.time() - start_time < timeout:
+            try:
+                if self.is_server_running():
+                    return True
+                if (
+                    self.process_frontend.poll() is not None
+                    or self.process_prefiller.poll() is not None
+                    or self.process_decoder.poll() is not None
+                ):
+                    self._close_log_files()
+                    return False
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)